tests/rptest/tests/partition_balancer_test.py

```diff
@@ -540,7 +544,9 @@ def test_fuzz_admin_ops(self):
         self.start_consumer(1)
         self.await_startup()
 
-        admin_fuzz = AdminOperationsFuzzer(self.redpanda, min_replication=3)
+        admin_fuzz = AdminOperationsFuzzer(self.redpanda,
+                                           min_replication=3,
+                                           retries=0)
```

Comment:
Not sure that every admin op will work without retries. Let's do a run with a few dozen retries.

Reply:
It fails with the retries too, I guess that the retries help with some % of failures but overall we treat the symptoms and hide the root cause. The idea of this PR is to make the test less probabilistic to highlight the main problem and then treat it.

I suspect that the real problem is that we expect `rpk` to work when a minority of the nodes are down but it isn't the case. Once it's confirm we should rise an issue to the rpk team and then either wait for a fix or ditch `rpk` and use lower level API which gives us more control in choosing a node we communicate with. Maybe we may stick with `rpk` but do not pass a failed node as part of the brokers list.

Reply:
I've run [a hundred](https://buildkite.com/redpanda/redpanda/builds/20679#01858370-77a8-4b4f-ad70-7fcf56b5fcc5) and they all passed :tada: